app/controllers/patient_admin/signups_controller.rb

```diff
@@ -0,0 +1,19 @@
+class PatientAdmin::SignupsController < PatientAdmin::ApplicationController
+  skip_before_action :authenticate_patient_admin_patient_user!, only: :basic
+
+  before_action :set_user
+
+  def basic
```

Comment:
Let's try to to build RESTful architecture, and if possible use only standard controller actions - new/index/edit/show/create/update/destroy.